app/components/forms/wizard/other-details-step.js

```diff
@@ -0,0 +1,160 @@
+import Component from '@ember/component';
+import { later } from '@ember/runloop';
+import { observer, computed } from '@ember/object';
```

Comment:
Codacy found an issue: ['observer' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)